common/src/main/java/bisq/common/util/GcUtil.java

```diff
@@ -59,20 +60,25 @@ private static void maybeReleaseMemory(long trigger) {
         long preGcMemory = Runtime.getRuntime().totalMemory();
         if (preGcMemory > trigger * 1024 * 1024) {
             System.gc();
+            totalInvocations++;
             long postGcMemory = Runtime.getRuntime().totalMemory();
-            log.info("GC reduced memory by {}. Total memory before/after: {}/{}. Took {} ms.",
+            long duration = System.currentTimeMillis() - ts;
+            totalGCTime += duration;
+            log.info("GC reduced memory by {}. Total memory before/after: {}/{}. Took {} ms. Total GC invocations: {} / Total GC time {} sec",
                     Utilities.readableFileSize(preGcMemory - postGcMemory),
                     Utilities.readableFileSize(preGcMemory),
                     Utilities.readableFileSize(postGcMemory),
-                    System.currentTimeMillis() - ts);
-            if (DevEnv.isDevMode()) {
+                    duration,
+                    totalInvocations,
+                    totalGCTime / 1000d);
+           /* if (DevEnv.isDevMode()) {
                 try {
                     // To see from where we got called
                     throw new RuntimeException("Dummy Exception for print stacktrace at maybeReleaseMemory");
                 } catch (Throwable t) {
                     t.printStackTrace();
                 }
-            }
+            }*/
```

Comment:
If commented out, better to remove it completely.

Reply:
I left it as its still helpful for memory/performance profiling to see from where we call it too often...
But Ok to me to remove it as well...